### PR TITLE
AQA tests for OCD-2533-Update eCQM numbers and versions for 2019 repo…

### PR DESCRIPTION
**Pull request by ashwinimore**
…rting bundle

### CODE REVIEW

**Review by kekey1 — APPROVED** [review]

File: src/test/java/gov/healthit/chpl/aqa/pageObjects/BasePage.java [hunk 1]
```diff
@@ -21,4 +21,13 @@ public static WebElement mainContent(final WebDriver driver) {
         element = driver.findElement(By.id("main-content"));
         return element;
     }
+
+    /**
+     * Return CQM Accordion on listing details page or edit CP page.
+     * @param driver WebDriver
+     * @return CQM Accordion element
+     */
+    public static WebElement cqmAccordion(final WebDriver driver) {
```
Comment (andlar) [inline src/test/java/gov/healthit/chpl/aqa/pageObjects/BasePage.java]: I don't think this should be part of the BasePage object. Not every page has this element, while they all should have the "main-content" one. My guess is that the only things in the basepage should be that main-content and navigation header/footer stuff

File: src/test/java/gov/healthit/chpl/aqa/pageObjects/ListingDetailsPage.java [hunk 2]
```diff
@@ -9,30 +9,30 @@
...
+
     /**
      * Returns element that holds Accessibility Standard Process Text.
      * @param driver WebDriver
      * @return Text element
      */
     public static WebElement accessibilityStandardText(final WebDriver driver) {
-        element = driver.findElement(By.xpath("//*[@id=\"panel-certification-criteria\"]/div[54]/div/div/div/table/tbody/tr[3]/td[2]/ul/li[1]"));
-        return element;
+        return driver.findElement(By.xpath("//*[@id=\"panel-certification-criteria\"]/div[54]/div/div/div/table/tbody/tr[3]/td[2]/ul/li[1]"));
```
Comment (andlar) [inline src/test/java/gov/healthit/chpl/aqa/pageObjects/ListingDetailsPage.java:24]: Is this targeting a specific criteria in the list of them? Will it work on a listing that doesn't attest to that criteria? Or maybe on one that only attests to 3 or 4 criteria? I see it's looking for the 54th something...

File: src/test/resources/Features/OCD-2533-eCQMNumberandVersionUpdate.feature [hunk 3]
```diff
@@ -0,0 +1,29 @@
+@Regression
+Feature: OCD-2533 -  Update eCQM numbers and versions for 2019 reporting bundle
+
+Scenario Outline: Verify newly added CQMs in listing details
+    Given I am on listing details page of listing with ID "<DB_ID>"
+    When I look at CQM details
+    Then I see newly added CQMs "<CMSID_title>"
```
Comment (andlar) [inline src/test/resources/Features/OCD-2533-eCQMNumberandVersionUpdate.feature:7]: Did the ticket require adding new CQMs to a particular set of listings? If not, how can you be sure the listings in your scenarios will have these CQMs? Or are you just checking that they're in the list of "available" ones, without checking that the Listing attests to either?

**Review by andlar — CHANGES_REQUESTED** [review]

File: src/test/resources/Features/OCD-2533-eCQMNumberandVersionUpdate.feature (hunk 3 above)
Comment (ashwinimore) [reply src/test/resources/Features/OCD-2533-eCQMNumberandVersionUpdate.feature:7]: The latter. I am opening list of all criteria and CQM to just view that newly added values show on listing details (not attested to).

File: src/test/java/gov/healthit/chpl/aqa/pageObjects/ListingDetailsPage.java (hunk 2 above)
Comment (ashwinimore) [reply src/test/java/gov/healthit/chpl/aqa/pageObjects/ListingDetailsPage.java:24]: This element was not used in the CQM test. I'll review and update it as necessary in next PR. I removed all 'element =' and added return to element definition to all elements in this page object file, that's why this shows in changes.

Comment (andlar) [reply src/test/java/gov/healthit/chpl/aqa/pageObjects/ListingDetailsPage.java:24]: Ahh, I'd missed that it wasn't really a change in functionality

**Review by andlar — APPROVED** [review]

**Review by iyerashwin14 — APPROVED** [review]

**Pull request merged**